packages/devtools_app/lib/src/shared/server/_analytics_api.dart

```diff
@@ -71,15 +71,10 @@ Future<String?> getConsentMessage() async {
 
 /// Confirm with package:unified_analytics that the consent message
 /// has been shown to the user.
-Future<bool> confirmConsentMessageShown() async {
+Future<void> confirmConsentMessageShown() async {
   if (isDevToolsServerAvailable) {
-    final resp = await request(apiSetConsentMessageAsShown);
-    if (resp?.statusOk ?? false) {
-      return true;
-    }
+    await request(apiAnalyticsConsentMessageShown);
```

Comment:
same with name here. I think I was confused before thinking that this API would also be used to get a value and not just set it. Having the name make it clear that it is only a one-way api (set only) would be good.